.github/workflows/ci-nightly.yaml

```diff
@@ -1,7 +1,7 @@
 name: Kata Containers Nightly CI
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '0 */2 * * *'
```

Comment:
Would the other option be to add `workflow_dispatch`, so we can manually build it whenever, or does that impact the trigger path too much?

Reply:
That's a very good idea, Steve!
Let me see if I can easily do that.

Reply:
Done!